src/utils/paths.rs

```diff
@@ -20,6 +20,7 @@ pub const HASHMAP: [&'static str; 5] = ["std", "collections", "hash", "map", "Ha
 pub const HASH: [&'static str; 2] = ["hash", "Hash"];
 pub const IO_PRINT: [&'static str; 3] = ["std", "io", "_print"];
 pub const LINKED_LIST: [&'static str; 3] = ["collections", "linked_list", "LinkedList"];
+pub const MEM_FORGET: [&'static str; 3] = ["core", "mem", "forget"];
```

Comment:
Yep. Not because of the core thing (if you're using core its already Advanced (probably unsafe) Rust and there's not much use for this lint), but because it's known as `mem::forget` anyway.